### ISSUE TITLE
Unable to split document into lower levels

### ISSUE THREAD

**Issue by plevold** [owenh000/asciidoctor-multipage#5]
No matter what i set `multipage-level` to I always get a document that looks like this:
![image](https://user-images.githubusercontent.com/10157556/106732774-6337a380-6611-11eb-91bc-101b75e8f69a.png)

I want to Split the document into pages for the second level sections (the `===` sections). Here's the source:

```asciidoc
= Sample Document
Author Name
:doctype: book
:toc: center
:toclevels: 3
:multipage-level: 3
:sectnums:

= Part 1

Intro

== Level 2

Some text goes here

=== Level 3

other text goes here

== Other Level 2

Some more text goes here

=== Level 3

additional text goes here
```

Built using the following command: `asciidoctor -r asciidoctor-multipage -b multipage_html5 -D out sample.adoc`

Am I doing something wrong?

I'm using the following gem versions:
```
asciidoctor (2.0.12, 2.0.10)
asciidoctor-multipage (0.0.3)
```

**Comment by owenh000** [comment]
Thanks for the report! This is fixed in commit 3b70976d44eff040e91732b972f934b18a61eefc. The fix is included in release v0.0.4.

**Comment by plevold** [comment]
Worked like a charm. Thanks!
